lib/y2lan_restart_common.pm

```diff
@@ -166,7 +166,7 @@ sub check_network_status {
         die "$expected_status was not any of the known expected status! Check journalctl findings \n$journal_findings\n\n";
     }
     clear_journal_log();
-    type_string "\n\n";    # make space for better readability of the console
+    enter_cmd "\n";    # make space for better readability of the console
```

Comment:
I think for this particular case, `type_string` is more readable, with `enter_cmd` one of new line characters is sort of hidden.

Reply:
How about

```suggestion
    send_key 'ret' for 1..2;    # make space for better readability of the console
```

Reply:
I don't like the `for` variant.

Reply:
As it's not a single place, we could introduce some wrapper method which puts multiple new lines. As `type_string` will remain in testapi I would just keep it the way it is. Or the plan is to get rid of `type_string` completely? As we still use it e.g. in text fields.

Reply:
@asdil12 you should really get accustomed to a simple for loop :)

`type_string` will stay.

I will keep the original code with `type_string "\n\n"`